src/notifier/keep_alive_monitor.py

```diff
@@ -74,18 +97,19 @@ def check_last_keep_alive(self):
             events = []
             for service in self._last_keep_alive.keys():
                 seconds_since_last = (datetime.now() - self._last_keep_alive[service]).seconds
-                logging.debug(f"Keep-alive check for {service.name}: Last activity {seconds_since_last} seconds ago.")
-                if seconds_since_last > self._last_keep_alive_threshold_seconds[service]:
+                threshold = self._last_keep_alive_threshold_seconds[service]
+                logging.debug(f"Keep-alive check for {service}: Last activity {seconds_since_last}s/{threshold}s")
+                if seconds_since_last > threshold:
                     message = (
-                        f"Your harvester appears to be offline! "
+                        f"Your {service.name} appears to be offline! "
                         f"No events for the past {seconds_since_last} seconds."
```

Comment:
Here we should probably add some helpful hints, e.g.

> No events for the past 320 seconds. You can adjust your config.yaml to either increase the notify threshold from {current threshold}, or disable the checks for this service by removing {service.name} from the list of `monitored_services`.

Reply:
But maybe this verbosity should only be included in the `logging.warning` below and not the actual notification.

Reply:
I think it doesn’t hurt including in both.

Reply:
I've also been thinking that "offline" might not be the best term here. We could instead phrase it as "not healthy" since offline has a strong connotation that the service isn't running, when in reality in most cases it's is sort of online, just wedged somehow. :D 

If it's rephrased as healthiness, then the messaging could be something like `Your {service.name} is unhealthy and hasn't behaved as expected for the past {seconds_since_last} seconds. (This check can be adjusted.)` .. which hopefully would lead anyone that is annoyed by it to go read documentation on how to not monitor that service, or to adjust the thresholds.

Reply:
Updated wording, LMK if that looks good.